OpenQA/Qemu/Proc.pm

```diff
@@ -187,6 +188,7 @@ sub configure_blockdevs {
         $size .= 'G' if defined($size);
 
         if (defined $backing_file) {
+            $backing_file = File::Spec->rel2abs($backing_file) unless File::Spec->file_name_is_absolute($backing_file);
```

Comment:
You can omit the `unless` because `rel2abs` will leave absolute paths as-is:
```suggestion
            $backing_file = File::Spec->rel2abs($backing_file);
```

The same counts for the other occurrences.